yt/frontends/_skeleton/data_structures.py

```diff
@@ -143,7 +148,8 @@ def _parse_parameter_file(self):
         #   self.geometry  <= a lower case string
         #                     ("cartesian", "polar", "cylindrical"...)
         #                     (defaults to 'cartesian')
-        pass
+        self.current_time = -1  # required, change this !
```

Comment:
It might be worth suggesting what a good value would be for this in a code that doesn't have time dependence. Since it is required, a helpful suggestion might be good here.

Reply:
Good point. This didn't occur to me. Actually I don't know why it's required at all. I should look for the reason first.
Do you think ˋ0.0` would make sense here ?